backend/metadata_writer/src/metadata_writer.py

```diff
@@ -325,8 +333,14 @@ def is_kfp_v2_pod(pod) -> bool:
 
                     output_artifacts = []
                     for name, art in argo_output_artifacts.items():
-                        artifact_uri = argo_artifact_to_uri(art)
-                        if not artifact_uri:
+                        artifact_uri_check = argo_artifact_to_uri(art)
```

Comment:
Can we just call this `artifact_uri`?

Reply:
Yes we can, it's just a habit of not overwriting variables while making some checks.